core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaStore.java

```diff
@@ -74,7 +71,7 @@
   private final int initTimeout;
   private final int timeout;
   private final Seq<Broker> brokerSeq;
-  private final ZkClient zkClient;
+  private final ZkUtils zkUtils;
```

Comment:
A user that is using the schema registry and wants to migrate to a secure cluster will probably need to authenticate the client here before setting the ZK secure ACL. Otherwise this client won't be able to create znodes. I think it mostly reads, but there are instances of writes to ZK.